### ISSUE TITLE
fix: adjust colors for hover / pressed interaction states meeting AA

### ISSUE THREAD

**Issue by coraliefeil** [solid-design-system/solid#546]
- [x] Double check all components for compliant color contrast
- [x] Research if wcag 2.2 leads to requirements needing to be implemented
- [x] Research if hover state - grey background in dropdowons etc. has to meet any color contrast

Info:

Does an element’s hover state have to reach 3:1 contrast ratio?
**Question**
I have a dropdown list with black text in a form. Does the background bar on the hover states as you hover over each selection option require a 3.1 contrast ratio against the white container background, or can I use a light gray background bar behind each selection for the hover states (such as #E6E6E6)?
**Answer**
Under the WCAG Success Criterion 1.4.11: Non-text Contrast, the gray bar that shows the hover state is not required to reach 3:1 contrast with the white background. One reason for this is that the position of the pointer device, such as the mouse cursor, also helps visually indicate which element has hover.
**Link**

**Comment by coraliefeil** [comment]
Closing this issue. All done. All good.
FYI @karlbaumhauer @yoezlem
